packages/apps-engine/src/definition/messages/ISystemMessage.ts

```diff
@@ -0,0 +1,79 @@
+import type { IMessage } from '.';
+
+export type SystemMessageType =
```

Comment:
It would be very cool if we could add some comments explaining each of the types of system message. But I'm not sure we have this information available, right? Maybe where you sourced these types there is some explanation?

Reply:
Unfortunately there is none, but I'll add a few comments explaining some of those, I guess it would be better than nothing